l10n/de_DE/LC_MESSAGES/civioffice.po

```diff
@@ -456,10 +598,281 @@ msgstr ""
 "Renderer konvertiert die angegebene Datei zu diesem MIME-Typ, z.B. "
 "\"application/pdf\"."
 
-#: api/v3/CiviOffice/Convert.php
-msgid "Live Snippets"
-msgstr "Live-Snippets"
-
 #: api/v3/CiviOffice/Convert.php
 msgid "Contents for tokens referring to configured Live Snippets."
 msgstr "Inhalte für Platzhalter konfigurierter Live-Snippets."
+
+#: civioffice.php
+msgid "Create CiviOffice document"
+msgstr "CiviOffice-Dokument erstellen"
+
+#: civioffice.php
+msgid "Access CiviOffice"
+msgstr "CiviOffice-Zugriff"
+
+#: civioffice.php
+msgid "Create documents using CiviOffice."
+msgstr "Dokumente mit CiviOffice erstellen."
+
+#: civioffice.php
+msgid "CiviOffice Settings"
+msgstr "CiviOffice-Einstellungen"
+
+#: civioffice.php
+msgid "CiviOffice"
+msgstr "CiviOffice"
+
+#: civioffice.php
+msgid "Upload Documents"
+msgstr "Dokumente hochladen"
+
+#: civioffice.php
+msgid "Available Tokens"
+msgstr "Verfügbare Tokens"
+
+#: templates/CRM/Civioffice/Form/DocumentFromSingleContact.tpl
+#: templates/CRM/Civioffice/Form/Task/CreateDocumentsTrait.tpl
+msgid "Activity"
+msgstr "Aktivität"
+
+#: templates/CRM/Civioffice/Form/DocumentRenderer/Settings/LocalUnoconv.tpl
+msgid "Please provide the path of your Unoconv binary."
+msgstr "Bitte stellen Sie den Pfad zu unoconv bereit."
+
+#: templates/CRM/Civioffice/Form/DocumentRenderer/Settings/LocalUnoconv.tpl
+msgid ""
+"We strongly recommend creating a system-wide lock file on this server to "
+"synchronise access. (It may only run one unoconv process at the same time.) "
+"Such a file is created automatically at the location shown below. Please "
+"note: This path needs to be equal in every civicrm environment on this "
+"server. Otherwise, locking is only active for this very civicrm instance!"
+msgstr ""
+"Wir empfehlen eindringlich eine System-weite Sperrdatei zu erstellen, um den "
+"Zugriff zu synchronisieren. (Es kann nur jeweils ein unoconv Prozess zur "
+"gleichen Zeit laufen.) Die Datei wird automatisch am unten angezeigten Ort "
+"erstellt. Bitte beachten: Der Pfad muss auf allen CiviCRM-Umgebungen auf "
+"diesem Server gleich sein. Anderenfalls ist das Sperren nur für diese "
+"CiviCRM-Instanz aktiv."
+
+#: templates/CRM/Civioffice/Form/DocumentRenderer/Settings/LocalUnoconv.tpl
+msgid ""
+"Run the DOCX document through the converter prior to processing in order to "
+"optimise/repair possibly corrupted CiviCRM tokens in the document."
+msgstr ""
+"DOCX-Dokumente vor der Verarbeitung durch den Konverter laufen lassen, um "
+"sie zu optimieren und mögliche korrupte CiviCRM-Tokens zu reparieren."
```

Comment:
Arguing for the plural of *Token* being *Token* in German, not *Tokens*, or let's just use *CiviCRM-Platzhalter*.

Reply:
Yes, there should be no *s* at the end. As *Token* is also used in the CiviCRM translation I think we should keep that word, even though I'd prefer *Platzhalter*.